podman/api/uds.py

```diff
@@ -89,6 +89,8 @@ def connect(self) -> None:
 class UDSConnectionPool(urllib3.HTTPConnectionPool):
     """Specialization of HTTPConnectionPool for holding UNIX domain sockets."""
 
+    # pylint complains if attributes aren't all caps.  Nobody will ever fix this.
+    # pylint: disable=invalid-name
     ConnectionCls = UDSConnection
```

Comment:
```suggestion
    ConnectionCls = UDSConnection     # pylint: disable=invalid-name
```
Would be less invasive. Otherwise, I believe you are turning off checking for the remainder of the file.

Reply:
Thanks, fixed and will rebase and force-push.